notion-core/src/test/scala/com/ambiata/notion/core/LocationIOSpec.scala

```diff
@@ -186,17 +189,17 @@ class LocationIOSpec extends Specification with ScalaCheck { def is = s2"""
     } yield tail must_== all
   }.set(maxSize = 10)
 
-  def sha1LikeMundane = prop { (loc: LocationTemporary, linesNumber: NaturalIntSmall) =>
-    val lines = (0 to linesNumber.value).toList.map("line"+_)
+  def sha1LikeMundane = prop { (loc: LocationTemporary, linesNumber: PositiveIntSmall) =>
+    val lines = (1 to linesNumber.value).toList.map("line"+_)
     for {
       path <- LocalTemporary(loc.path).file
       l    =  LocalLocation(path.path)
       i    <- loc.io
       _    <- i.writeUtf8Lines(l, lines)
-      s1   <- i.reduceLinesUTF8(l, LineReducer.sha1)
+      s1   <- i.reduceBytes(l, BytesReducer.sha1)
       s2   <- Checksum.file(path, SHA1).map(_.hash)
-    } yield s1 must_== s2
-  }.set(maxSize = 10)
+    } yield s1 ==== s2
+  }.set(maxSize = 10, workers = 1)
```

Comment:
Any reason for the `workers`? If we can, we should avoid forcing serial tests IMO.